windows/src/desktop/help/build.sh

```diff
@@ -77,7 +77,7 @@ build_hhc_header() {
   echo '
 <HTML>
   <HEAD>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=windows-1252" />
```

Comment:
Why is this changing from utf-8 to windows-1252?

Reply:
The .hhc file is used by the HTML Help Workshop / HTMLHelp which does not have good support for Unicode in its index. Unfortunately, there are not any great replacements for the .chm file format at present, apart from going with fully web-based documentation (whether stored locally or on help.keyman.com).